ovos_core/intent_services/__init__.py

```diff
@@ -98,6 +80,28 @@ def __init__(self, bus, config=None):
         self.bus.on('intent.service.padatious.manifest.get', self.handle_padatious_manifest)
         self.bus.on('intent.service.padatious.entities.manifest.get', self.handle_entity_manifest)
 
+    def _load_pipeline_plugins(self):
+        # TODO - replace with plugin loader from OPM
+        self.adapt_service = AdaptService(config=self.config.get("adapt", {}))
```

Comment:
Instance variables defined outside of init here. Consider also moving these to properties with deprecation warnings since not all plugins will have references like this

Reply:
done